### ISSUE TITLE
Is Firebase integration with Geospatial Creator in Unity actually possible?

### ISSUE THREAD

**Issue by Akash-Elnoorkar** [google-ar/arcore-unity-extensions#179]
Hello,

I am trying to integrate firebase into my project which is having Geospatial Creator present in it. But it always shows these errors whenever I try to build the APK. I have done debugging and testing on it as well. I first tried importing Firebase into an empty project by only enabling Firebase Analytics and the APK got built successfully. Then, I tried importing Geospatial Creator to this project and it gave me the errors present. Also, tried the vice-versa procedure and still throwed the same erorrs.

Any information on this would be very helpful and useful. Thanks.

![Screenshot (81)](https://github.com/google-ar/arcore-unity-extensions/assets/142652286/e93f0de5-83f4-469c-8fdb-4f99b98aa21c)
![Screenshot (82)](https://github.com/google-ar/arcore-unity-extensions/assets/142652286/94d7ea1a-28bb-49e5-97ba-f55535fd1b61)
![Screenshot (83)](https://github.com/google-ar/arcore-unity-extensions/assets/142652286/8d8688c3-1768-488d-be68-fe5f9e2d40ef)
![Screenshot (84)](https://github.com/google-ar/arcore-unity-extensions/assets/142652286/6d940dde-6031-4edd-94af-28f02f7c9a9f)
![Screenshot (85)](https://github.com/google-ar/arcore-unity-extensions/assets/142652286/b63402a6-8946-49bf-bcfe-3d1a38f093be)
![Screenshot (86)](https://github.com/google-ar/arcore-unity-extensions/assets/142652286/b6272b5f-5201-4dad-9bdf-43979c26aba0)
![Screenshot (87)](https://github.com/google-ar/arcore-unity-extensions/assets/142652286/24f05465-547c-4f9b-9e92-64bb69f039e2)
![Screenshot (88)](https://github.com/google-ar/arcore-unity-extensions/assets/142652286/fa5a4cd7-16bf-444d-a522-8e0328d238ee)
![Screenshot (89)](https://github.com/google-ar/arcore-unity-extensions/assets/142652286/13005362-6e61-40f1-99c0-fac031e82371)
![Screenshot (90)](https://github.com/google-ar/arcore-unity-extensions/assets/142652286/a252ba8c-f1ef-4fbd-9f1f-ffde0f1a4e99)

**Comment by devbridie** [comment]
Does this work for you? [Gradle Build fails [Android] 'META-INF/kotlinx-coroutines-core.kotlin_module'](https://stackoverflow.com/questions/50774577/gradle-build-fails-android-meta-inf-kotlinx-coroutines-core-kotlin-module)

I'm seeing that error message in your fourth screenshot.

**Comment by Akash-Elnoorkar** [comment]
Nope, this doesn't work. I am actually shocked to see this. I really can't understanc why firebase and geospatial can't work together, I guess I have to develop my own REST API for this.

**Comment by burakcetn** [comment]
You should use the Without EDM4U version of Arcore Extension. https://developers.google.com/ar/develop/unity-arf/getting-started-extensions#without-edm4u-compatibility
